Device/Device/schema.json

```diff
@@ -0,0 +1,199 @@
+{
+    "namespace": "Device",
+    "name": "Device",
```

Comment:
This change apply for both attributes or  just to name attribute?
We think about a namespace to represent a category of problem domain.

For example: Thing:Device, Environment:Air:AirQualityObserved (the namespace separator ":" is just a suggestion may / or _ can be a better choice).